kate/recovery/src/com.rs

```diff
@@ -1,39 +1,106 @@
-use std::{convert::TryInto, ops::Range};
+use std::{collections::HashMap, convert::TryInto, ops::Range};
 
+use anyhow::{anyhow, Result};
 use dusk_bytes::Serializable;
 use dusk_plonk::{fft::EvaluationDomain, prelude::BlsScalar};
 
 // TODO: Constants are copy from kate crate, we should move them to common place
 pub const DATA_CHUNK_SIZE: usize = 31;
 const PADDING_TAIL_VALUE: u8 = 0x80;
 
-// Reconstructs app extrinsics from extrinsics layout and data
+pub struct MatrixDimensions {
+	pub rows: usize,
+	pub cols: usize,
+	pub chunk_size: usize,
+}
+
+fn map_cells(
```

Comment:
Just a quick docstring to describe what this fn does would be nice. Doesn't have to be more than one line.

Reply:
Done.